src/components/Article.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from 'react-router-dom'
+import {useEffect, useState} from 'react'
+import axios from 'axios'
+
+function Article({articles}){
+
+    const [article, setArticle] = useState([])
+    const [loading, setLoading] = useState(true)
+    const [error, setError] = useState('')
+
+
+    const {article_id} = useParams()
+    useEffect(()=>{
+        axios.get(`https://northcoders-news-app-ei5k.onrender.com/api/articles/${article_id}`)
+        .then(({data}) => {
+            setLoading(false)
```

Comment:
As this is still read in thread, I'd advise setting loading and error states after you've set your data state.